attributes/default.rb

```diff
@@ -25,27 +25,27 @@
 default['ndb']['group']                               = node['install']['user'].empty? ? "mysql" : node['install']['user']
 default['ndb']['connectstring']                       = ""
 
-default['ndb']['DataMemory']                          = "50"
-# Calculate IndexMemory size by default, can be overriden by user.
-default['ndb']['IndexMemory']                         = ""
+default['ndb']['DataMemory']                          = "98"
```

Comment:
why 98?

Reply:
98M is the default value since 7.6.2 (the M is in `config.ini.erb`) - plus 50M runs out fast especially if you keep a VM running for a month.